default_public_container_registries.txt

```diff
@@ -0,0 +1 @@
+quay.io
```

Comment:
Do we really need a file for that?
In that case, I think this file should be moved together with `default_ocp_versions` to a new `data` directory as @filanov suggested.

Reply:
How do you propose we avoid the file? `generate_ocp_version` is supposed to generate `onprem-environment` and `onprem-iso-fcc.yaml` based off of those values. Until now, it did it off environment variables - we shouldn't generate files based on environment variables, we should generate files based on other files.

Reply:
@YuviGold @filanov what do you say? Can you lgtm?